examples/getting_started/pt/nvflare_lightning_getting_started.ipynb

```diff
@@ -335,10 +335,10 @@
     "from src.lit_net import LitNet\n",
     "\n",
     "from nvflare.app_common.workflows.fedavg import FedAvg\n",
-    "from nvflare.app_opt.pt.job_config.base_fed_job import BaseFedJob\n",
+    "from nvflare.app_opt.pt.job_config.base_fed_job import PTBaseFedJob\n",
```

Comment:
So, are there `PTFedJob` and `TFFedJobs` as well? Maybe we can skip the "Base" in the name in these training framework specific job classes.

Reply:
We currently don't have PTFedJob/TFFedJobs. We could remove the "Base" in these names, would this class naming still make sense? 

FedJob -> BaseFedJob -> PTFedJob

Reply:
Another proposal: Does this: PTJob -> CommonJob -> FedJob make sense?

Maybe we don’t need to make the names suggest the parent class they inherit from.